### PR TITLE
Adding unit tests for Serverless API Lambda build and fix issue with documentation

### PR DESCRIPTION
**Pull request by mxro**
For #125 

- Adding unit test for Serverless API Lambda build
- Fixing build issue with documentation

### CODE REVIEW

File: workspaces/docs/docs/templates/serverless-api/index.md [hunk 1]
```diff
@@ -119,20 +119,24 @@ This works very well in combination with secrets for GitHub actions.
...
 {
   "platform": "node"
 }
 ```
-This object will be used for build proccess on every serverless function
+
+This object will be used for build process on every serverless function
 
 If you want to change esbuild config for specific function, just add esbuild config, like this:
-```sh
```
Comment (mxro) [inline workspaces/docs/docs/templates/serverless-api/index.md:130]: `sh` file type not supported yet for generating this type of documentation.

**Comment by dimkk** [comment]
Sorry, for inconvenience, I will put more time in investigating testing pipeline and docs next time.

**Comment by mxro** [comment]
No worries at all, I think there is some more configuration required to make the build pipelines work in forked projects!

Also just came across another issue in the end-to-end tests, https://github.com/goldstack/goldstack/runs/5435465573?check_suite_focus=true#step:6:1120 which I think is somekind of mixup with package versions I will try to investigate. But cannot release the new version yet until that is solved!

**Comment by mxro** [comment]
I think I found the issue with the `utils-sh` package - I think it was not published because the original merge to master failed and the PR with the fix didn't trigger the library publish (since no files in `utils-sh` were changed). Running the build now with a forced changed to `utils-sh` which should hopefully resolve this error!

**Pull request merged**